src/python/WMCore/WMRuntime/Monitors/NATSMonitor.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python
+
+"""
+_NATSMonitor_
+"""
+from __future__ import division
+
+# system modules
+import logging
+import traceback
```

Comment:
no need to import traceback. Just use `logging.exception` instead (there are a few cases below)

Reply:
ok, done